.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
```

Comment:
I'll remove it but I tend to disagree. By enforcing the editor to insert the correct end-of-line the need for making git do a conversion can be dropped.